tests/integration/components/conversations/conversation-thread-test.js

```diff
@@ -60,7 +61,7 @@ test('it delays rendering head until loaded', function(assert) {
   assert.equal(page.conversationParts().count, 0, 'No part is rendered.');
 });
 
-test('it renders each conversation part, sorted by insertedAt', function(assert) {
+test('it renders each conversation part', function(assert) {
```

Comment:
I'm not sure we wanted to change this test, but change `conversationParts` to be `conversation.sortedConversationParts`, no?

Reply:
In fact, it makes me wonder if we shouldn't pass `conversationParts` in at all, and instead the `conversation-thread` component should take the `conversation` only and make use of its `sortedConversationParts` internally.

Reply:
The problem is we are using plain objects and not ember data models.  So we don't have access to `sortedConversationParts`.  lmk if you see something different.

Also if we don't pass in `sortedConversationParts`, `didUpdateAttrs` will never fire b/c none of it's direct attrs changed when something comes through a web socket.

Reply:
You can mock the `conversation` by having a key of `sortedConversationParts`.

But the latter point seems like the key. I'm curious to hear what @begedin thinks about this, as well.

Reply:
Yeah lmk what you you would like me to do.  By mocking out sortedConversationParts, we are def closer to the real implementation.  But not actually testing the real implementation.

Reply:
I think in that case, we should pass in `sortedConversationParts` (as we do in the actual uses of this component) and maybe even change the component to use a `sortedConversationParts` (vs naming it just `conversationParts`), that way it's clear that we're always passing in something that needs sorted. Right now this test leads us to think the component is used differently than it actually is _and_ that there isn't any date sorting. We need to clarify that the sorting happens outside the component's scope.